### PR TITLE
[ALLUXIO-2147]Add javadoc for getFileInfoTest

### PR DESCRIPTION
**Pull request by onesmileapp**
https://alluxio.atlassian.net/browse/ALLUXIO-2147
mf1633045

### CODE REVIEW

**Comment by jsimsa** [comment]
Jenkins, ok to test

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
@onesmileapp Thanks! Please sign the CLA agreement.

**Comment by aaudiber** [comment]
alluxio-bot, check this please

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 1]
```diff
@@ -311,21 +311,22 @@ public void getFileInfo() throws Exception {
     long fileId;
     FileInfo info;
 
-    fileId = mFileSystemMaster.getFileId(ROOT_URI);
+    fileId = mFileSystemMaster.getFileId(ROOT_URI);//get the ROOT_URI's id
```
Comment (aaudiber) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: Please place the comments on the lines before what they comment on.

Comment (jsimsa) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: I think the purpose of this task was to add a method-level Javadoc ... I don't find these new comments helpful or necessary

Comment (aaudiber) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: +1, the method-level javadoc should describe what this test is testing

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 2]
```diff
@@ -304,7 +304,9 @@ public void getFileId() throws Exception {
     Assert.assertEquals(IdUtils.INVALID_FILE_ID,
         mFileSystemMaster.getFileId(NESTED_FILE_URI.join("DNE")));
   }
-
+  /**
```
Comment (gpang) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: I think there should be an empty line before this line.

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 3]
```diff
@@ -321,7 +323,8 @@ public void getFileInfo() throws Exception {
     Assert.assertEquals(NESTED_URI.getPath(), info.getPath());
     Assert.assertEquals(NESTED_URI.getPath(), mFileSystemMaster.getFileInfo(NESTED_URI).getPath());
 
-    fileId = mFileSystemMaster.getFileId(NESTED_FILE_URI);
+
```
Comment (gpang) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: these two lines do not have to be modified.

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 4]
```diff
@@ -321,6 +324,7 @@ public void getFileInfo() throws Exception {
     Assert.assertEquals(NESTED_URI.getPath(), info.getPath());
     Assert.assertEquals(NESTED_URI.getPath(), mFileSystemMaster.getFileInfo(NESTED_URI).getPath());
 
+
```
Comment (gpang) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: This extra blank line should be removed.

File: core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java [hunk 5]
```diff
@@ -304,7 +304,10 @@ public void getFileId() throws Exception {
     Assert.assertEquals(IdUtils.INVALID_FILE_ID,
         mFileSystemMaster.getFileId(NESTED_FILE_URI.join("DNE")));
   }
-
+
```
Comment (gpang) [inline core/server/src/test/java/alluxio/master/file/FileSystemMasterTest.java]: remove the trailing spaces on this line.

**Comment by gpang** [comment]
@onesmileapp There are some trailing whitespace on one of your new lines. Thanks!

**Pull request merged**